### PR TITLE
docs: RFC 0003 Standardize Github Labels

### PR DESCRIPTION
**Pull request by CalvinRodo**
See document.

### CODE REVIEW

File: rfcs/xxxx-standard-gh-labels.md
Comment (dj2) [inline rfcs/xxxx-standard-gh-labels.md:20]: nit: lables -> labels

File: rfcs/xxxx-standard-gh-labels.md
Comment (dj2) [inline rfcs/xxxx-standard-gh-labels.md:27]: What do you mean by 'copied into github'? We'll create a fork of the labels repo above and they'll be housed there? Or, a discussion?

File: rfcs/xxxx-standard-gh-labels.md
Comment (dj2) [inline rfcs/xxxx-standard-gh-labels.md:30]: nit: add .

File: rfcs/xxxx-standard-gh-labels.md
Comment (dj2) [inline rfcs/xxxx-standard-gh-labels.md:41]: I think your ```'s are in the wrong spots. Putting 3 before as "```yaml" and 3 at the very end should syntax highlight I think

File: rfcs/xxxx-standard-gh-labels.md
Comment (dj2) [inline rfcs/xxxx-standard-gh-labels.md:62]: I don't think we should propose this. We probably need security issues to _not_ go through the public trackers as they become public but to follow a separate security procedure. This may make it confusing to folks who think will then think security issues should get filed through these trackers.

File: rfcs/xxxx-standard-gh-labels.md
Comment (dj2) [inline rfcs/xxxx-standard-gh-labels.md:79]: nit: change must to MUST ? same with one below

File: rfcs/xxxx-standard-gh-labels.md
Comment (dj2) [inline rfcs/xxxx-standard-gh-labels.md:86]: final sentence cut off.

File: rfcs/xxxx-standard-gh-labels.md
Comment (dj2) [inline rfcs/xxxx-standard-gh-labels.md:111]: Delete these sections if they aren't needed.

File: rfcs/xxxx-standard-gh-labels.md
Comment (dj2) [inline rfcs/xxxx-standard-gh-labels.md:106]: The one potential drawback is folks may find this constricting in that labels are top down. I believe this is mitigated because there is the escape hatch of product/project specific labels in the proposal.

File: rfcs/xxxx-standard-gh-labels.md
Comment (dj2) [inline rfcs/xxxx-standard-gh-labels.md:109]: The current alternative is what we have now which is some mix of default Github labels and ad-hoc project labels. The downside is getting a holistic view of all the projects and understanding their current status along with the cognitive load of switching projects requiring learning a new set of labels.

File: rfcs/xxxx-standard-gh-labels.md
Comment (dj2) [inline rfcs/xxxx-standard-gh-labels.md:2]: (Delete the stuff in ()'s I put in in as a comment should have noted to delete them, heh)

File: rfcs/xxxx-standard-gh-labels.md
Comment (dj2) [inline rfcs/xxxx-standard-gh-labels.md:1]: I'd say number it 0003 as that would be the next sequential one. And, even if we reject, still good to keep it numbered in some fashion I think?

File: rfcs/xxxx-standard-gh-labels.md
Comment (CalvinRodo) [reply rfcs/xxxx-standard-gh-labels.md:106]: Yeah that was the plan with BU's being able to extend is that it allows us to have a standard view into all projects so we can start to build up some automated reporting for senior management and to address common tasks across projects.

Comment (dj2) [reply rfcs/xxxx-standard-gh-labels.md:106]: Yup, I think listing it as a potential drawback but give the reasoning for why we don't see it being an issue is beneficial to future readers to see that we thought about it but provided our arguments against the drawback.

File: rfcs/xxxx-standard-gh-labels.md
Comment (CalvinRodo) [reply rfcs/xxxx-standard-gh-labels.md:27]: I mean a github repo will be created to manage this action, I'll clarify text

File: rfcs/xxxx-standard-gh-labels.md
Comment (CalvinRodo) [reply rfcs/xxxx-standard-gh-labels.md:20]: fixed

File: rfcs/xxxx-standard-gh-labels.md
Comment (CalvinRodo) [reply rfcs/xxxx-standard-gh-labels.md:41]: Good catch the backticks were just to format nicely but formatting as yaml makes more sense.

File: rfcs/xxxx-standard-gh-labels.md
Comment (CalvinRodo) [reply rfcs/xxxx-standard-gh-labels.md:62]: That makes sense but at the same time we have shared some security issues in our repos but mainly before things have made it to production. 

https://github.com/cds-snc/covid-alert-server/issues?q=is%3Aissue+label%3ACCCS+is%3Aclosed+label%3Asecurity

So I think there might be a need for this but I'm open to more discussion around it.

File: rfcs/xxxx-standard-gh-labels.md
Comment (CalvinRodo) [reply rfcs/xxxx-standard-gh-labels.md:86]: Fixed, it was a sentence I meant to make shorter.

File: rfcs/xxxx-standard-gh-labels.md
Comment (CalvinRodo) [reply rfcs/xxxx-standard-gh-labels.md:109]: Good alternative I've added it to the doc

File: rfcs/xxxx-standard-gh-labels.md
Comment (CalvinRodo) [reply rfcs/xxxx-standard-gh-labels.md:1]: Works for me

File: rfcs/xxxx-standard-gh-labels.md
Comment (CalvinRodo) [reply rfcs/xxxx-standard-gh-labels.md:2]: will do

**Review by dj2 — APPROVED** [review]
Looks good to me.

**Comment by CalvinRodo** [comment]
I'd like to share this with the PM Community since it has impacts on them.

**Comment by dj2** [comment]
Yup, as was mentioned in the dev meeting today, we should probably let this sit for ~2 weeks before we land. So, I guess my approval is just, I approve, we need to wait to see if anyone disapproves during the intervening time.

**Comment by wmoussa-cds** [comment]
https://github.com/cds-snc/docs/pull/12#issuecomment-810616058

File: rfcs/xxxx-standard-gh-labels.md
Comment (dj2) [reply rfcs/xxxx-standard-gh-labels.md]: Maybe we just need to set the description to something like 'Non-production security issues' ? Then we have the SECURITY.md files which point to where to send production security issues.

File: rfcs/0003-standard-gh-labels.md
Comment (wmoussa-cds) [inline rfcs/0003-standard-gh-labels.md:87]: too soon for this proposed line, I suggest we don't dive into ownership outside dev co

File: rfcs/0003-standard-gh-labels.md
Comment (wmoussa-cds) [inline rfcs/0003-standard-gh-labels.md:89]: we can leave it for the product tech lead to kick this off, and mention eventually the product manager owns it

File: rfcs/0003-standard-gh-labels.md
Comment (wmoussa-cds) [inline rfcs/0003-standard-gh-labels.md:94]: @maxneuvians what do you think?

File: rfcs/0003-standard-gh-labels.md
Comment (wmoussa-cds) [inline rfcs/0003-standard-gh-labels.md:100]: I don't think it is about dictating and we don't want to dictate anything. The drawback is some product will use the new convention and some not which at the end won't allow us to reach the maximum benefit of this RFC. 

We will have to sit with the teams not using the labels and understand their concerns. 

What do you think?

Comment (wmoussa-cds) [reply rfcs/0003-standard-gh-labels.md:100]: But I think if you offer OLA compliant labels out of the box and easy integration, this will be attractive enough

File: rfcs/0003-standard-gh-labels.md
Comment (timarney) [inline rfcs/0003-standard-gh-labels.md:66]: Would suggest an a11y label be included here `baseline`

File: rfcs/0003-standard-gh-labels.md
Comment (CalvinRodo) [reply rfcs/0003-standard-gh-labels.md:89]: Good idea, I'll update the RFC to focus on Dev community only.

File: rfcs/0003-standard-gh-labels.md
Comment (CalvinRodo) [reply rfcs/0003-standard-gh-labels.md:87]: Makes sense will update

File: rfcs/0003-standard-gh-labels.md
Comment (CalvinRodo) [reply rfcs/0003-standard-gh-labels.md:66]: Agreed.

File: rfcs/0003-standard-gh-labels.md
Comment (CalvinRodo) [reply rfcs/0003-standard-gh-labels.md:87]: Changing to head of development.

File: rfcs/0003-standard-gh-labels.md
Comment (dj2) [reply rfcs/0003-standard-gh-labels.md:66]: Can we call it `accessibility` instead of `a11y`. Numbered words aren't understandable unless you have context.

File: rfcs/xxxx-standard-gh-labels.md
Comment (CalvinRodo) [reply rfcs/xxxx-standard-gh-labels.md]: Updated Drawbacks

File: rfcs/0003-standard-gh-labels.md
Comment (CalvinRodo) [reply rfcs/0003-standard-gh-labels.md:69]: yeah I used the longform for it since https://mobilea11y.com/blog/inaccessible-a11y/

**Comment by wmoussa-cds** [comment]
Thanks @CalvinRodo for suggesting and sorry for the delay.

**Comment by dj2** [comment]
As an FYI, the CDS-SNC default repository settings are updated to use the labels presented in this RFC. Projects created after this point should have the new labels by default.

**Comment by wmoussa-gc** [comment]
@dj2 is there a plan to migrating existing repo to this new set of labels?

**Comment by dj2** [comment]
@CalvinRodo is setting up a github action that will apply these labels automatically. We won't apply them to old repos without help from the dev team impacted, as I believe you need to do some work to migrate.

**Pull request merged**
